src/main/java/com/metamx/metrics/SysMonitor.java

```diff
@@ -141,7 +144,9 @@ public void emit(ServiceEmitter emitter)
       if (mem != null) {
         final Map<String, Long> stats = ImmutableMap.of(
             "sys/mem/max", mem.getTotal(),
-            "sys/mem/used", mem.getUsed()
+            "sys/mem/used", mem.getUsed(),
+            "sys/mem/actual/used", mem.getActualUsed(),
```

Comment:
Buffers are almost never used compared to page cache. I don't think those will be different.